lib/geo.js

```diff
@@ -278,4 +286,3 @@ function geoDistance(x1, y1, x2, y2, options) {
 
   return 2 * Math.asin(f) * EARTH_RADIUS[type];
 }
-
```

Comment:
sorry - I didn't do that directly... atom must of... and I didn't catch it when reviewing the diff for the commit.